torch/_inductor/virtualized.py

```diff
@@ -72,6 +72,7 @@ def __init__(self):
         super().__init__()
         self.removed_buffers = set()
         self.inplaced_to_remove = set()
+        self.index_dtype = "tl.int64"
```

Comment:
This may result in a perf regression in indexing-compute-heavy kernels? Although, your benchmark run suggests it doesn't.

Reply:
I think this code is only hit in unit tests when `V.kernel` is set to `NullKernel`.  In codegen it should point to a real kernel, which has this value set via analysis.